walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,144 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.Set;
+import java.util.HashSet;
 
 public final class FindMeetingQuery {
+  private final int AVAILABLE = 0;
+  private final int NOT_AVAILABLE = 1;
+  private final int MINUTES_IN_A_DAY = 60 * 24;
```

Comment:
There is an equivalent constant in tests file. I can merge them in another common file if needed.

Reply:
Same as above.